src/sentry/processing/realtime_metrics/project_budget_pb2.py

```diff
@@ -0,0 +1,32 @@
+# Generated by the protocol buffer compiler.  DO NOT EDIT!
```

Comment:
we should not be checking in generated files

usually the way these are distributed is through a client library generated from the protobuf definitions in the service itself (such that we would add `yourservice-client` as a dependency)